react-to-do/src/components/ToDoContainer.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import Header from "./Header";
+import TodosList from "./ToDoLists";
+class TodoContainer extends React.Component {
+  state = {
+    todos: [
+      {
+        id: 1,
+        title: "Setup development environment",
+        completed: true,
+      },
+      {
+        id: 2,
+        title: "Develop website and add content",
+        completed: false,
+      },
+      {
+        id: 3,
+        title: "Deploy to live server",
+        completed: false,
+      },
+    ],
+  };
+
+  handleChange = (id) => {
+    this.setState((prevState) => ({
+      todos: prevState.todos.map((todo) => {
+        if (todo.id === id) {
+          return {
+            ...todo,
+            completed: !todo.completed,
+          };
+        }
+        return todo;
+      }),
+    }));
+  };
+
+  delTodo = (id) => {
+    this.setState({
+      todos: [
+        ...this.state.todos.filter((todo) => {
+          return todo.id !== id;
+        }),
+      ],
+    });
+  };
+
+  render() {
+    return (
+      <React.Fragment>
+        <Header />
+        <TodosList
+          todos={this.state.todos}
+          handleChangeProps={this.handleChange}
+          deleteTodoProps={this.delTodo}
+        />
+      </React.Fragment>
+    );
+  }
+}
```

Comment:
This part may cause the problem. You can try to add your InputToDo class inside return.